torch/_dynamo/mutation_guard.py

```diff
@@ -61,40 +61,40 @@ def custom_setattr(self, key, value):
 
 
 class GenerationTracker:
-    generation = 0
-    dynamic_classes = ExactWeakKeyDictionary()
-    generation_values = ExactWeakKeyDictionary()
+    generation: int = 0
+    dynamic_classes: ExactWeakKeyDictionary
+    generation_values: ExactWeakKeyDictionary
 
     @classmethod
-    def tag(cls, obj):
+    def tag(cls, obj: Any) -> None:
         cls.generation_values[obj] = cls.generation
 
     @staticmethod
-    def mark_class_dynamic(cls):
+    def mark_class_dynamic(cls: type) -> None:
```

Comment:
```suggestion
    def mark_class_dynamic(cls: Type[torch.nn.Module]) -> None:
```
Need to double check if this is valid for subclasses, but it looks like it is.